spark/v3.2/spark/src/jmh/java/org/apache/iceberg/spark/source/IcebergSourceBenchmark.java

```diff
@@ -187,4 +202,45 @@ protected void withTableProperties(Map<String, String> props, Action action) {
       restoreProperties.commit();
     }
   }
+
+  protected void writePosDeletes(CharSequence path, long numRows, double percentage) throws IOException {
+    OutputFileFactory fileFactory = newFileFactory();
+    SparkFileWriterFactory writerFactory = SparkFileWriterFactory.builderFor(table())
+            .dataFileFormat(fileFormat())
```

Comment:
Iceberg uses 4 spaces for continued indentation, not 8.

Reply:
This method isn't needed any more. They are moved to class `IcebergSourceDeleteBenchmark`. I will delete them.